src/view/about.twig

```diff
@@ -0,0 +1,25 @@
+{% extends "base-template.twig" %}
+{% block title %}: "About"{% endblock %}
+{% block content %}
+
+<div class="container">
+  <div class="mx-5">
+    <div class="px-3 pb-3 bg-light">
+      <h2 class="fs-3 mb-5">"About"</h2>
```

Comment:
Good question! When the translation component is there (and I don't recall the exact syntax for it) it will become
`{% "About" | trans %}`. I'm denoting that it's a message stingfrom the .po file.